ts/screens/profile/analytics/mixpanel/mixpanelAnalytics.ts

```diff
@@ -1,7 +1,17 @@
 import { mixpanelTrack } from "../../../../mixpanel";
+import { updateMixpanelProfileProperties } from "../../../../mixpanelConfig/profileProperties";
+import { GlobalState } from "../../../../store/reducers/types";
 import { FlowType, buildEventProperties } from "../../../../utils/analytics";
 
-export function trackMixpanelSetEnabled(value: boolean, flow: FlowType) {
+export async function trackMixpanelSetEnabled(
+  value: boolean,
```

Comment:
I just left the previous name. But we could change it. It's the `value` of MP enabling (`true`, `false`). In this case I didn't refactor it because the the meaning is already present in the method name.

```ts
export async function trackMixpanelSetEnabled(
  enabled: boolean,
```

or 

```ts
export async function trackMixpanelSetEnabled(
  isEnabled: boolean,
```

Any other suggestion? What do you think?

Reply:
In my opinion, the first option is preferable.

Reply:
Both are perfectly fine, but if I remember correctly in the app we use isEnabled most frequently

Reply:
Okay, then. Done [here](https://github.com/pagopa/io-app/pull/5857/commits/51a03851444bb73d1a8ff215e97a01aa3426e783).